billing/test/lib/ucan-stream.js

```diff
@@ -24,14 +24,60 @@ export const test = {
 
     const shard = randomLink()
 
-    /** @type {import('../../lib/api.js').UcanReceiptMessage[]} */
+    /**
+     * @type {import('../../lib/api.js').UcanReceiptMessage<[
+     *   | import('@web3-storage/capabilities/types').BlobAllocate
+     *   | import('@web3-storage/capabilities/types').BlobRemove
+     *   | import('@web3-storage/capabilities/types').StoreAdd
+     *   | import('@web3-storage/capabilities/types').StoreRemove
+     * ]>[]}
+     */
     const receipts = [{
       type: 'receipt',
       carCid: randomLink(),
       invocationCid: randomLink(),
       value: {
         att: [{
-          with: await randomDID(),
+          with: await randomDIDKey(),
+          can: 'web3.storage/blob/allocate',
```

Comment:
Nit, could we use exported `.can`  from capabilities repo instead?

Reply:
I did reach for that first but then I realised I didn't do it for the rest of them in the other tests and it would be a bigger change. Worth doing in retrospect.